packages/EasyCore/src/Bridge/Symfony/Resources/config/services.yaml

```diff
@@ -6,3 +6,6 @@ services:
 
     EonX\EasyCore\Lock\LockServiceInterface:
         class: EonX\EasyCore\Lock\LockService
+
+    EonX\EasyCore\Bridge\Symfony\Messenger\ProcessWithLockMiddleware:
```

Comment:
Instead of having to type the service class twice, because the service id is basically the service class you can do that:

```yaml
EonX\EasyCore\Bridge\Symfony\Messenger\ProcessWithLockMiddleware: null
```

Reply:
Noted.